ouroboros-consensus-diffusion/CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ NOTE: version jumps from `0.13.0.0` to `0.15.0.0` because `0.14.0.0` was created
 - Updated `ouroboros-consensus-diffusion` to use `ouroboros-network-0.14.0.0`.
   `LowLevelRounNodeArgs` and `NodeKernel` records hold
   `PublicPeerSelectionState` variable.
+- Added `GetPeerSnapshot` block query
+
+### Non-Breaking
```

Comment:
Ah sorry for not saying this earlier: Can you revert the changes to this file, and instead add a [changelog fragment](https://github.com/IntersectMBO/ouroboros-consensus/blob/main/docs/website/contents/for-developers/ReleaseProcess.md#adding-a-changelog-fragment)?

Reply:
Sure, no problem